neo/Network/RPC/TransactionManager.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Neo.Cryptography.ECC;
+using Neo.Cryptography.ECC;
```

Comment:
Why this is changed but didn't cause travis to fail?

Reply:
Good question

Reply:
The file was with BOM and here it is removed, I don't know why it passed previously, but here is good.